includes/controls/media.php

```diff
@@ -271,9 +272,43 @@ public function content_template() {
 			<# } #>
 			<input type="hidden" data-setting="{{ data.name }}"/>
 		</div>
+		<div class="elementor-control-content elementor-control-media-size">
+			<div class="elementor-control-field elementor-control elementor-control-type-select">
+				<label for="elementor-control-default" class="elementor-control-title"><?php echo esc_html__( 'Image Size', 'elementor' ); ?></label>
+				<div class="elementor-control-input-wrapper media-size-input-wrapper">
+				<select data-setting="size" class="elementor-control elementor-control-type-select elementor-control-input-wrapper elementor-control-media-display-size select">
+				<?php
+				$sizes = $this->get_image_sizes_options();
+				foreach ( $sizes as $key => $value ) {
+					?>
+						<option value="<?php echo esc_attr( $key ); ?>" {{ data.controlValue.size == '<?php echo esc_attr( $key ); ?>' ? 'selected' : '' }}><?php echo esc_html( $value ); ?></option>
+					<?php
+				}
+				?>
+				</select>
+				</div>
+			</div>
+		</div>
+
 		<?php
 	}
 
+	protected function get_image_sizes_options() {
+		$sizes = get_intermediate_image_sizes();
+		$sizes[] = 'full';
+		$result = [];
+		foreach ( $sizes as $size ) {
+			$width = get_option( "{$size}_size_w" );
+			$height = get_option( "{$size}_size_h" );
+			if ( ! $width && ! $height ) {
+				$result[ $size ] = ucfirst( $size );
+			} else {
+				$result[ $size ] = ucfirst( esc_html__( $size, 'elementor' ) ) . " ($width x $height)"; // phpcs:ignore WordPress.WP.I18n.NonSingularStringLiteralText
```

Comment:
Is it translated properly?
coz using a non-string in this method usually doesn't work (expect for some edge cases)

Reply:
Do we have a solution for this issue?

Reply:
Removed